### DISCUSSION TITLE
Add Support For Multilingual Spellchecking

### DISCUSSION THREAD

**Discussion by MichaelDevon** [standardnotes/forum#3397] [General]
SN has a big community of users from the EU, and i think the lack of multi-language enabled spell checking support is a problem that EVERY non native English user has faced. There must be a spell checking library that can be added to have this "feature", because as it stands right now, its functionally useless for half of the world.

**Comment by effieeee** [comment]
Hi again Michael! While spell checking with multiple languages isn't available on the mobile and web platforms, the desktop app does have its own spellcheck library where you can select languages to be spellchecked:
![image](https://github.com/standardnotes/forum/assets/54989748/27554445-d5cc-468d-a8c4-4b82374a06ae)

**Reply by MichaelDevon** [reply]
Thank you! I'd never would have found it! Are there any plans to add this library to mobile?

**Reply by effieeee** [reply]
No plans for this on mobile yet I'm afraid, but we'll try to keep that in mind!
